### ISSUE TITLE
Outpost Supply Quest (Repeat bug)

### ISSUE THREAD

**Issue by JIFCode** [EdenServer/community#158]
I registered an outpost supply quest I had already finished, Gustaberg region, and upon hand in the key item is not being removed. I am able to talk to Shigezane I.M. and it registers the quest as being finished, sound, text, yet the key item ( Gustaberg Supplies ) persists on both my characters.

**Comment by JigokunoOusama** [comment]
I experienced the same issue today.  Ronfaure supplies are stuck with me until tally, I guess.

**Comment by bsgilber** [comment]
`Me too` -- IGN Deceit

Unlocked Ronfaure OP, accidentally clicked on supply mission again, received temp KI for Ronfaure. I went to turn them in again, but the temp KI remained in inventory despite the correct interaction with the OP guard. 

I waited 24HRs (happened to coincide with weekly tally) but was unable to get any other OP supplies; tried to turn them in again and had the same correct interaction with the OP guard but and kept the temp KI.

Asked for help in Discord, @godmode said if it was not resolved in another day (by 12-10-18) then he will reset it manually.
